### PR TITLE
Fixes #13546 - Update tftp to true

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by stbenjam** [comment]
I would guess it should probably go to a capsule migration too then, but it's still up in the air what we enable by default.

One question I've had about these scenarios, how is this working in an upgrade case? It sounds like we'll be flipping TFTP on for users who explicitly turned it off in previous versions of Katello, right? If it can't be avoided in the move to the new installer, it needs to be prominently documented.

**Comment by ehelms** [comment]
@stbenjam can you re-review this?

File: config/capsule.migrations/160425201608-tftp-true.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Change value to true
+
+answers['foreman_proxy'] = {
+  'tftp' => yes,
```
Comment (ehelms) [inline config/capsule.migrations/160425201608-tftp-true.rb]: I think this should just be true.

File: config/katello.migrations/160328095934-tftp-true.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Change value to true
+
+answers['foreman_proxy'] = {
+  'tftp' => yes,
```
Comment (ehelms) [inline config/katello.migrations/160328095934-tftp-true.rb]: I think this should just be true.

**Comment by stbenjam** [comment]
Not sure we actually need this now that I think about it. It is on by default in foreman proxy: https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/params.pp#L186

**Comment by chris1984** [comment]
@stbenjam care if i close this PR then?

**Comment by stbenjam** [comment]
Nope, go ahead! Thanks @chris1984

**Pull request closed without merging**